packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2535,28 +2534,14 @@ function completeUnitOfWork(unitOfWork: Fiber): void {
   // sibling. If there are no more siblings, return to the parent fiber.
   let completedWork: Fiber = unitOfWork;
   do {
-    if (revertRemovalOfSiblingPrerendering) {
+    if (__DEV__) {
       if ((completedWork.flags & Incomplete) !== NoFlags) {
-        // This fiber did not complete, because one of its children did not
-        // complete. Switch to unwinding the stack instead of completing it.
-        //
-        // The reason "unwind" and "complete" is interleaved is because when
-        // something suspends, we continue rendering the siblings even though
-        // they will be replaced by a fallback.
-        // TODO: Disable sibling prerendering, then remove this branch.
-        unwindUnitOfWork(completedWork);
-        return;
-      }
-    } else {
-      if (__DEV__) {
-        if ((completedWork.flags & Incomplete) !== NoFlags) {
```

Comment:
Can you please leave these else branches? At least for now while the code paths are still relatively new.

Reply:
Oh wait never mind you did leave this else branch. Ignore me 😆